content/docs/communities/moderation/_index.md

```diff
@@ -0,0 +1,114 @@
++++
+title = "Community Moderation"
+weight = 100
+[extra]
+emoji = "🛡️"
+tile = "I want to deploy moderation tools"
+last_updated = "2022-10-27T08:00:00Z"
++++
+
+## Power levels
+
+Matrix has a basic system of roles, sometimes also called Power Levels. It is a
+number usually going from 0 to 100. Advanced users can decide to use a different
+range, but we will not cover it here for the sake of simplicity.
+
+By default, there are three roles in Matrix:
+
+| Power Level | Role          |
+|-------------|---------------|
+| 0           | User          |
+| 50          | Moderator     |
+| 100         | Administrator |
+
+By default:
+
+- **Users** can send messages, media, reactions and redact their own messages in
+  a room.
+- **Moderators** can also change the room name, address, topic, remove users
+  from the room (temporarily or permanently), redact other's messages, send a
+  noisy notification to everyone in the room
+- **Administrators** can also change the history visibility (whether people can
+  see messages from before they joined or not), enable encryption in the room,
+  remove entire servers from the room, and promote others as Moderators or
+  Administrators
+
+Most people in your community are just going to be regular users. Usually
+Moderators are appointed to handle the ad-hoc moderation issues. Note that this
+can be an issue if you want to preserve the anonymity of your Trust & Safety
+team for their own security. The moderation bot mjolnir brings a good answer to
```

Comment:
This sounds quite alarming, and no extra context is given about what this loss of anonymity actually means.

Reply:
Well it is alarming because it is :)

If the moderation team is not anonymous, they can be doxxed, harassed, and for more extreme cases even assaulted. Retaliation can happen and is a very real threat. I'm going to look for resources to provide more context.